logzio-sender/src/main/java/io/logz/sender/LogzioSender.java

```diff
@@ -165,35 +163,12 @@ public void drainQueueAndSend() {
 
     public void send(JsonObject jsonMessage) {
         // Return the json, while separating lines with \n
-        enqueue((jsonMessage+ "\n").getBytes());
-    }
-
-    private void enqueue(byte[] message) {
-        if (isEnoughDiskSpace()) {
-            logsBuffer.enqueue(message);
-        }
-    }
-
-    private boolean isEnoughDiskSpace() {
-        if (dontCheckEnoughDiskSpace) {
-            return true;
-        }
-
-        int actualUsedFsPercent = 100 - ((int) (((double) queueDirectory.getUsableSpace() / queueDirectory.getTotalSpace()) * 100));
-        if (actualUsedFsPercent >= fsPercentThreshold) {
-
-            reporter.warning(String.format("Logz.io: Dropping logs, as FS used space on %s is %d percent, and the drop threshold is %d percent",
-                    queueDirectory.getAbsolutePath(), actualUsedFsPercent, fsPercentThreshold));
-
-            return false;
-        } else {
-            return true;
-        }
+        logsBuffer.enqueue((jsonMessage+ "\n").getBytes());
```

Comment:
BTW: We have a bug here, since String.getBytes() encodes the string to bytes using default platform charset. We, on the listener side assume bytes are encoded using UTF-8. This should be "UTF-8".